### ISSUE TITLE
docker doesn't work

### ISSUE THREAD

**Issue by Ana06** [openSUSE/osem#2345]
Following the _docker-compose_ section in [our contribution guide](https://github.com/openSUSE/osem/blob/master/CONTRIBUTING.md#development-environment), I run `docker-compose run --rm osem bundle exec rake db:bootstrap` and I got the following error:

```
** Execute db:bootstrap
rake aborted!
ActiveRecord::NoDatabaseError: FATAL:  database "osem_development" does not exist
```
Shouldn't docker create the database for me? :thinking:

**Comment by Ana06** [comment]
This was a bug in Ruby 2.5.0, but if I get the docker again it install Ruby 2.6

**Comment by hennevogel** [comment]
fixed in 02f0fb078c3fe2dd22cafa379fb6da8426cc06d8

**Comment by Ana06** [comment]
Thanks!
